protocols/request-response/Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["network-programming", "asynchronous"]
 async-trait = "0.1"
 cbor4ii = { version = "0.3.2", features = ["serde1", "use_std"], optional = true }
 futures = { workspace = true }
-instant = "0.1.13"
+web-time = { workspace = true }
```

Comment:
Actually `web-time` (and `instant`) is never used in request-response, so we can just remove `instant`.
```suggestion
```